demos/quickstart/cdn_loader/index.html

```diff
@@ -0,0 +1,63 @@
+    </script>
+</head>
+
+<body onresize="doResize()">
+    <div id="placeholder">
```

Comment:
If you inline the placeholder styles here then the style tag in the `head` isn't needed. That would allow the head to be "instantiation" and the body to be "implementation"